qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -0,0 +1,462 @@
+                 model: str=None, **kwargs) -> None:
+        """
+        Args:
+            name: Name to use internally in QCoDeS
+            address: VISA ressource address
+            model: The model type, e.g. '2614B'
+        """
+        super().__init__(name, address, terminator='\n', **kwargs)
+
+        if model is None:
+            raise ValueError('Please supply Keithley model name, e.g.'
+                             '"2614B".')
+        knownmodels = ['2601B', '2602B', '2604B', '2611B', '2612B',
+                       '2614B', '2635B', '2636B']
+        if model not in knownmodels:
+            kmstring = ('{}, '*(len(knownmodels)-1)).format(*knownmodels[:-1])
+            kmstring += 'and {}.'.format(knownmodels[-1])
+            raise ValueError('Unknown model. Known model are: ' +
+                             kmstring)
+
+        self.model = model
+
+        self._vranges = {'2601B': [0.1, 1, 6, 40],
+                         '2602B': [0.1, 1, 6, 40],
+                         '2604B': [0.1, 1, 6, 40],
+                         '2611B': [0.2, 2, 20, 200],
+                         '2612B': [0.2, 2, 20, 200],
+                         '2614B': [0.2, 2, 20, 200],
+                         '2635B': [0.2, 2, 20, 200],
+                         '2636B': [0.2, 2, 20, 200]}
+
+        # TODO: In pulsed mode, models 2611B, 2612B, and 2614B
+        # actually allow up to 10 A.
+        self._iranges = {'2601B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 3],
+                         '2602B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 3],
+                         '2604B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 3],
+                         '2611B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 1.5],
+                         '2612B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 1.5],
+                         '2614B': [100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 0.01, 0.1, 1, 1.5],
+                         '2634B': [1e-9, 10e-9, 100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 10e-6, 100e-3, 1, 1.5],
+                         '2635B': [1e-9, 10e-9, 100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 10e-6, 100e-3, 1, 1.5],
+                         '2636B': [1e-9, 10e-9, 100e-9, 1e-6, 10e-6, 100e-6,
+                                   1e-3, 10e-6, 100e-3, 1, 1.5]}
+
+        # Add the channel to the instrument
+        channels = ChannelList(self, "Channels", KeithleyChannel,
+                               snapshotable=False)
+        for ch in ['a', 'b']:
+            ch_name = 'smu{}'.format(ch)
+            channel = KeithleyChannel(self, ch_name, ch_name)
+            channels.append(channel)
+            self.add_submodule(ch_name, channel)
```

Comment:
personally I would leave this out and make the channellist snapshotable. 
As this is you can do

```
key.smua.v
key.channels.smua.v
key.channels[0].v
```
I think it's more logical to only have access via channels